pkg/template/app/app.go

```diff
@@ -1,15 +1,17 @@
 package app
 
 import (
+	"fmt"
+
 	"github.com/giantswarm/apiextensions/v3/pkg/apis/application/v1alpha1"
 	applicationv1alpha1 "github.com/giantswarm/apiextensions/v3/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/k8smetadata/pkg/label"
+	"github.com/giantswarm/kubectl-gs/internal/key"
 	"github.com/giantswarm/microerror"
 	"github.com/spf13/afero"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/yaml"
-
-	"github.com/giantswarm/kubectl-gs/internal/key"
```

Comment:
This should remain in its own block as it's project internal - LGTM apart from that.